share/man/man4/puc.4

```diff
@@ -45,6 +45,35 @@ driver.
 The list of supported devices is in
 .Pa sys/dev/puc/pucdata.c .
 Support for new cards should be added there.
+.Sh HARDWARE
+The
+.Nm
+driver supports the following PCI serial and parallel ports:
```

Comment:
```suggestion
driver supports the following
PCI multi-port serial and parallel port controllers:
```
Since this one is a bit long, we have to make a decision about wrapping. The Roff language limits lines to 80 characters, and courtesy limits them to 72 when possible, so we want to be considerate of grouping semantic constructs together for grep.

Make sure we aren't loosing information while distilling. "multi-port" is an interesting detail.

Also consider that people tend to search for parallel ports as "parallel ports".

Even though it says so above, these aren't really ports. Controllers maybe?